lib/plugins/collaborators.js

```diff
@@ -102,26 +95,23 @@ module.exports = class Collaborators extends Diffable {
 
   remove (existing) {
     if (existing.pendinginvite) {
+      const data = Object.assign({ invitation_id: existing.invitation_id }, this.repo)
       if (this.nop) {
         return Promise.resolve([
-          new NopCommand(this.constructor.name, this.repo, this.github.repos.deleteInvitation.endpoint(
-            Object.assign(this.repo, { invitation_id: existing.invitation_id })), 'Delete Invitation')
+          new NopCommand(this.constructor.name, this.repo, this.github.repos.deleteInvitation.endpoint(data),
+            'Delete Invitation')
         ])
       }
-      return this.github.repos.deleteInvitation(
-        Object.assign(this.repo, { invitation_id: existing.invitation_id })
-      )
+      return this.github.repos.deleteInvitation(data)
     } else {
+      const data = Object.assign({ username: existing.username }, this.repo)
       if (this.nop) {
         return Promise.resolve([
-          new NopCommand(this.constructor.name, this.repo, this.github.repos.removeCollaborator(
-            Object.assign(this.repo, { username: existing.username })
-          ), 'Remove Collaborator')
+          new NopCommand(this.constructor.name, this.repo, this.github.repos.removeCollaborator(data),
```

Comment:
Be careful. This will remove the collaborators. See https://github.com/github/safe-settings/pull/342/files#diff-aadaae6765a1167f3187e283961920c01d5db933ff8a7908ff10554b5520e689R117.